server/routes/authRoutes.ts

```diff
@@ -43,21 +56,48 @@ export default function authRoutes(): Router {
     res.render('pages/one-login/enter-phone-number')
   })
 
+  post('/enter-phone-number', async (req, res, next) => {
+    res.redirect('/one-login/check-phone')
+  })
+
   get('/check-phone', async (req, res, next) => {
     res.render('pages/one-login/check-phone')
   })
 
   post('/check-phone', async (req, res, next) => {
-    res.render('pages/one-login/account-created')
+    res.redirect('/one-login/account-created')
   })
 
   get('/account-created', async (req, res, next) => {
     res.render('pages/one-login/account-created')
   })
 
+  post('/account-created', async (req, res, next) => {
+    res.redirect('/one-login/enter-email-address-login')
```

Comment:
It's not usable for a user to create and account and then have to login. Was this the route used by the rp prototype?

Reply:
I would expect the user to be "logged in" and session started behind the scenes and continue to /pop without additional input screens